diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt

```diff
@@ -25,7 +27,9 @@ import java.util.function.Consumer
 internal const val TEST_FILE_NAME = "TestFileName.kt"
 
 @Suppress("WRONG_WHITESPACE")
-private val log = LoggerFactory.getLogger({}.javaClass)
+private val log = KotlinLogging.logger(
```

Comment:
suggestion create an extension function for KotlinLogging which applies .initKtLintKLogger()

Also, why we do not use the following way of initialization: `KotlinLogging.logger {}.initKtLintKLogger()`?

Reply:
Funny thing - diktat complains on such expressions because of an empty block. I don't want to place a lot of new suppresses, we should maybe thing of some other way to allow these